### PR TITLE
Add external identifiers

### PR DESCRIPTION
**Pull request by justnero**
## Summary
Feature name might change before the actual release.

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/ExternalIdentifier.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    readOnly: true
+  externalId:
+    type: string
+    description: ID of the referenced entity within an external service.
+  createdTime:
+    $ref: ./CreatedTime.yaml
+  updatedTime:
+    $ref: ./UpdatedTime.yaml
+  _links:
+    type: array
```
Comment (noon-dawg) [inline openapi/components/schemas/ExternalIdentifier.yaml:34]: nitpick: this was displayed as a blank description in the rendered docs
```suggestion
  _links:
    description: Related links.
    type: array
```

File: openapi/paths/{resource}@{resourceId}@external-identifiers@{service}.yaml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+delete:
+  x-products:
+    - Users
+  tags:
+    - External identifiers
+  summary: Delete an external identifier
+  operationId: DeleteExternalIdentifier
+  x-sdk-operation-name: delete
+  description: |-
+    Deletes an external identifier with a specified resource, resource ID and a service name.
```
Comment (noon-dawg) [inline openapi/paths/{resource}@{resourceId}@external-identifiers@{service}.yaml:97]: nitpick: [serial comma](https://product.rebilly.dev/writing-style-guide/grammar-and-mechanics/commas/#serial-commas)

We also shouldn't add block scalers for single line descriptions ([line break guidance](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#line-breaks))

```suggestion
  description: Deletes an external identifier with a specified resource, resource ID, and service name.

**Review by noon-dawg — COMMENTED** [review]
This is looking great!

I added some very minor nitpicks

**Review by noon-dawg — DISMISSED** [review]

**Review by removed-account-123 — DISMISSED** [review]
Looks good to me :)

**Review by noon-dawg — DISMISSED** [review]

**Review by removed-account-123 — APPROVED** [review]

**Review by noon-dawg — APPROVED** [review]

**Review by 2e3s — APPROVED** [review]

**Pull request merged**